src/init/common.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <key.h>
+#include <logging.h>
+#include <pubkey.h>
+#include <random.h>
+
+#include <memory>
+
+static std::unique_ptr<ECCVerifyHandle> globalVerifyHandle;
+
+namespace init {
+void SetGlobals()
+{
+    // Initialize elliptic curve code
+    std::string sha256_algo = SHA256AutoDetect();
```

Comment:
unrelated 000e94b26c7340dfc834299d75848c34c73b3128: comment seems wrong? Maybe just remove it?